rules_gapic/bazel/src/main/java/com/google/api/codegen/bazel/ArgsParser.java

```diff
@@ -0,0 +1,57 @@
+package com.google.api.codegen.bazel;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+class ArgsParser {
+  private final Map<String, String> parsedArgs = new HashMap<>();
+
+  ArgsParser(String[] args) {
+    for (String arg : args) {
+      String[] argNameVal = arg.split("=");
```

Comment:
I think for this particular case I would do same even if there was no goal to avoid third-party dependencies (it is a simple call, and the tool does not do sophisticated comman line interface (only two args) so doing it manually justified).